app/src/main/java/wendu/jsbdemo/JsApi.java

```diff
@@ -14,66 +14,47 @@
 
 public class JsApi{
     @JavascriptInterface
-    public String testSyn(JSONObject jsonObject) throws JSONException {
-        return jsonObject.getString("msg") + "［syn call］";
+    public String testSync(JSONObject jsonObject) throws JSONException {
```

Comment:
i thought that we are already agree that we should always pass json format string,  This will make it look confusing in the future